src/cli/index.js

```diff
@@ -96,7 +96,7 @@ async function main(context) {
 
   const hasFilePatterns = context.filePatterns.length > 0;
   const useStdin =
-    !hasFilePatterns && (!process.stdin.isTTY || context.argv.filePath);
+    !hasFilePatterns && (!process.stdin.isTTY || context.argv.filepath);
```

Comment:
Out of scope bug fix which was actually causing issues in the if condition on line 105